point: 0
base_x1: 613
base_y1: 547
base_x2: 872
base_y2: 780
base_x3: 951
base_y3: 779
base_x4: 927
base_y4: 506
